pages/configuring_webif.rst

```diff
@@ -182,6 +182,31 @@ If you want to use nginx to proxy access to a Graylog server, you have several o
     }
   }
 
+Apache:
+------- 
+
+For the following samples we are assuming that your Graylog instance is running on ``graylog.internal.example.org`` using the default ports of 12900 for the REST API and 9000 for the web interface. SSL is disabled for both. You want to expose the Graylog web interface as ``https://graylog.example.org``. 
```

Comment:
Could you please emphasize that the Graylog server is running on the same host as Apache? Makes it clearer for people adapting the config.